README.md

```diff
@@ -8,6 +8,14 @@ Apache Flink is an open source stream processing framework with powerful stream-
 
 Learn more about Flink at [https://flink.apache.org/](https://flink.apache.org/)
 
+## Modules
+
+This repository contains the following modules
+
+* [Prometheus Connector](./prometheus-connector): Flink Prometheus Connector implementation; supports optional request signer
+* [Sample application](./example-datastream-job): Sample application showing the usage of the connector with DataStream API. It also demonstrates how to configure the request signer. 
```

Comment:
I don't think an integration test is useful for a user. And the lack of proper working examples is a common obstacle to the adoption from new to Flink.
This module is not supposed to be released in any form. Can't we just keep it as source, and possibly reference from the documentation?

Reply:
We have done it in tests previously to mitigate the "clutter" problem, and difficulty when creating the connector distribution jars.
https://github.com/apache/flink-connector-aws/tree/main/flink-connector-aws/flink-connector-dynamodb/src/test/java/org/apache/flink/connector/dynamodb/sink/examples

We could do something similar and point to them via README, OR we could add a maven profile that only adds the examples module for PR / nightlies

Reply:
I moved the example application under tests, as suggested, merging everything in a single class.
The usage of the AMP connector is commented out, being a separate dependency, but still present to show the usage, in case someone wants to use it.

https://github.com/apache/flink-connector-prometheus/blob/5afb890aa8fe860ce25bcf89f7cdee93a6d2c4be/flink-connector-prometheus/src/test/java/org/apache/flink/connector/prometheus/sink/examples/DataStreamExample.java